src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/StronglyTypedEnum_TestEnums.txt

```diff
@@ -273,6 +273,24 @@ Errors: Error 24-28: Invalid argument type (Enum (TestYeaNay)). Expecting a Enum
 >> TestXORYesNo( TestYesNo.No, TestYesNo.Yes )
 true
 
+>> If( false, TestYeaNay.Yea, false )
+TestYeaNay.Nay
+
+>> Text( If( false, TestYeaNay.Yea, false ) )
+"Nay"
+
+>> If( false, TestNumberCompareNumericCoerceFrom.V, 10 )
+TestNumberCompareNumericCoerceFrom.CalculatedOptionSetValue
+
+>> Text( If( false, TestNumberCompareNumericCoerceFrom.V, 10 ) )
+"CalculatedOptionSetValue"
+
+>> If( false, TestNumberCompareNumericCoerceFrom.V, 11 )
```

Comment:
> If( false, TestNumberCompareNumericCoerceFrom.V, 11 ) [](http://example.com/codeflow?start=2&length=54)

Is this test different than one on line 282?

Reply:
Added a comment.  10 is in the optionset, while 11 is not.  It is possible we could map the 10 to the option set value instead of using CalculatedOptionSetValue in the future.